src/main/java/frc/robot/swerve/ModuleIOReal.java

```diff
@@ -117,9 +105,9 @@ public double getAngle() {
     }
 
     @Override
-    public void setAngle(double angle) {
-        angleSetpoint = AngleUtil.normalize(angle);
-        Rotation2d error = new Rotation2d(angle).minus(new Rotation2d(currentAngle));
+    public void setAngle(Rotation2d angle) {
+        angleSetpoint = AngleUtil.normalize(angle.getRadians());
```

Comment:
> Instead of normalize you can simply use Java's [Math.floorMod](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#floorMod-int-int-)

Does it really matter that much which method I use?

> I do wonder, however, why is this needed at all?

I'll check if it works without it.

Reply:
> Does it really matter that much which method I use?

In this case, it does. If we can delete the `normalize()` function it means we have less code to read, test, and maintain.